python-sdk/tests_integration/test_example_dags.py

```diff
@@ -45,6 +45,7 @@ def session():
 MIN_VER_DAG_FILE: dict[str, list[str]] = {
     "2.3": ["example_dynamic_task_template.py", "example_bigquery_dynamic_map_task.py"],
     "2.4": ["example_datasets.py"],
+    "2.6.1": ["calculate_popular_movies_by_genre.py"],
```

Comment:
I am not sure what's the best way, but we might need to reach out to the release manager to request them to tag the release for the PR and include https://github.com/apache/airflow/pull/30943 in `2.6.1`. WDYT?

Reply:
Usually, I see Ephraim helping and taking care of releases. Might make sense to check with him once.

Reply:
Thanks a lot, @pankajkoti , he confirmed it will be included on 2.6.1 🎉